internal/compiler-bridge/src/main/scala/xsbt/API.scala

```diff
@@ -152,7 +155,7 @@ final class API(val global: CallbackGlobal) extends Compat with GlobalHelpers wi
           }
           val zincClassName = names.className
           val srcClassName = classNameAsString(symbol)
-          sourceVF foreach { source =>
+          sourceVF.foreach { source =>
```

Comment:
Was this done automatically? What's wrong with foreach being infix?

Reply:
It's distracting for some to be infix and others not, so I've just taken infix out of the picture.

Reply:
I am generally in favor of the trend towards using standard dot-and-paren invocation style, rather than aggressive infix style. However, `foreach` taking a lambda on RHS is a clear case that _should_ be infixed as the DSL is mimicking to be a language word, and the end result is not even dot-and-paren.

Reply:
One of the reason to always favour non-infix is that it makes it non-awkward to add something like `.headOption` at the end of it.  I guess with `foreach` that's different (if we ignore thinks like tap/pipe).  But outside of foreach I still stand by the consistency value of applying this universally.

I wouldn't be against you if you argued to the Scalafmt devs that `foreach` should be treated differently and got the default changed, or even made it as an option that you then preferred to have switched on.